src/main/scala/scalajsGames/GameHolder.scala

```diff
@@ -45,14 +45,23 @@ class GameHolder(canvasName: String, gameMaker: (Point, () => Unit) => Game) {
       ctx.font = "20pt Arial"
       ctx.textAlign = "center"
       ctx.fillText(message.get, bounds.x / 2, bounds.y / 2)
-      ctx.font = "14pt Arial"
-      ctx.fillText("Press any key to continue", bounds.x / 2, bounds.y / 2 + 30)
+      if (finalScore.isDefined) {
```

Comment:
Replace `isDefined/getOrElse` by 

```
finalScore match {
  case Some(score) => ...
  case None => ...
}
```

It is very rare that we use `Option#get`, most times, there is a better way.